dialogue-core/src/main/java/com/palantir/dialogue/core/VersionedTaggedMetricRegistry.java

```diff
@@ -114,9 +114,9 @@ public Counter counter(MetricName metricName, Supplier<Counter> counterSupplier)
     }
 
     @Override
-    public void addMetrics(String _safeTagName, String _safeTagValue, TaggedMetricSet _metrics) {
-        throw new UnsupportedOperationException(
-                "Operations involving transforming metricsets are not implemented as we don't need them in dialogue");
+    public void addMetrics(String safeTagName, String safeTagValue, TaggedMetricSet metrics) {
+        // Passed through without adding version tags
```

Comment:
is this just to minimize the LoC?

Reply:
It didn't seem as important for this metric and the change is already a tad complex, I can add versions here if you like.

Reply:
should be fine to revert this now right?

Reply:
Yep, will do